src/web-ui/src/public/Auth.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <secondary-layout>
+    <div v-if="showingSignUp" class="sign-up-notice">
+      <p>A verification code will be sent to the email address entered.</p>
+      <p>Passwords must contain at least 8 characters, including an uppercase letter, a lowercase letter, a special character, and a number.</p>
+    </div>
+    <AmplifyAuthenticator :authConfig="authConfig" ref="authenticator" />
+  </secondary-layout>
+</template>
+
+<script>
+import { components, AmplifyEventBus } from 'aws-amplify-vue';
+import SecondaryLayout from '../components/SecondaryLayout/SecondaryLayout.vue';
+
+
+export default {
+  name: 'Auth',
+  components: {
+    AmplifyAuthenticator: components.Authenticator,
+    SecondaryLayout
+  },
+  data() {
+    return {
+      showingSignUp: undefined,
+      authConfig: {
+        signInConfig: {
+          header: 'Sign In'
+        },
+        signUpConfig: {
+          hideAllDefaults: true,
+          header: 'Create account',
+          signUpFields: [
+            {
+              label: 'Email',
+              key: 'email',
+              type: 'email',
+              required: true
+            },
+            {
+              label: 'Password',
+              key: 'password',
+              type: 'password',
+              required: true
+            },
+            {
+              label: 'Username',
+              key: 'username',
+              type: 'string',
+              required: true
+            },
+          ]
+        }
+      }
+    }
+  },
+  mounted() {
+    this.$refs['authenticator'].$watch('displayMap', (newVal) => {
```

Comment:
Another nit pick - dot notation would be ideal here. As it stands, might mess with IDE's renaming feature down the line if needed.